.github/workflows/actions_build.yml

```diff
@@ -82,12 +82,11 @@ jobs:
         ${{ matrix.coverage && '-Pcoverage' || '' }} \
         ${{ matrix.leak && '-Pleak' || '' }} \
         -PnoLint \
+        -PflakyTest=off \
```

Comment:
nit: How about using `true` and `false`?
I'm not strong on this so `on` and `off` are also find to me. 😄

Reply:
Either is fine with me. `true` and `false` sounds better.